### DISCUSSION TITLE
Clear Api Key

### DISCUSSION THREAD

**Discussion by bstar** [jackMort/ChatGPT.nvim#324] [Q&A] [answered]
I generated a ChatGPT api key last week that never worked.  I think services were down and made it seem like the key was bad.  I ultimately deleted and generated a couple new ones to test.

Anyway, I can't clear this old key in ChatGPT.nvim, I always get a message when running :ChatGPT - "incorrect api key provided".  I've tried different api_key_cmd values and I've tried ENV variables.  Nothing changes this old key.  

Is there an easy way to clear this?  I've cleared cache and the .local nvim dir, to no effect.

**Comment by jackMort** [comment]
It may be just a response in a history, try to ask another question or open new session: Ctrl+n

**Comment by bstar** [comment]
> It may be just a response in a history, try to ask another question or open new session: Ctrl+n

I thought that might be the case too, but it just hangs when I try to ask a question...
https://imgur.com/a/LCcMDk9

**Comment by jackMort** [comment, marked as answer]
you need to submit in `normal` mode (`esc` and then `enter`) or by `ctrl+enter`

**Reply by bstar** [reply]
Thanks so much for taking the time to respond, this is indeed the problem.  Not sure how missed this detail.
